test/object_store/run

```diff
@@ -75,12 +75,16 @@ storage_opts = format_map({'type': 'S3',
                            'bucket': s3_public_bucket})
 ks = 'test_ks'
 cf = 'test_cf'
+rows = [('0', 'zero'),
```

Comment:
If walking this route, then it should become random ~~set~~ dict of random ~~values~~ tuples

Reply:
that'd change the nature of this test. it was a deterministic test in the sense that the dataset is predefined, if we want to use a randomized data, this test would be a randomized test. i'd prefer to do one thing at a time. if we want to go with a randomized test, how about implementing it in a separate pull request?

Reply:
It was deterministic just because it's author (me) was too lazy to put there anything other than one-two-three :(

Reply:
okay. will change it in a follow-up change. i have some more pull requests on this test in my local branch actually.